db/migrate/20220527124109_create_likees.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLikees < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likees do |t|
+      t.string :name
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+
+      t.timestamps
```

Comment:
- [ ] Kindly check these references too for the same reason. to replicate the error on your side, kidly drop your database and re-run the migrations.